assets/js/components/setup-wizard/wizard-progress-step.js

```diff
@@ -62,12 +63,12 @@ class WizardProgressStep extends Component {
 		}
 
 		return (
-			<div className={ `
-				googlesitekit-wizard-progress-step
-				googlesitekit-wizard-progress-step--${ step }
-				${ currentStep ? 'googlesitekit-wizard-progress-step--current' : '' }
-				googlesitekit-wizard-progress-step--${ stepKey }
-			` }>
+			<div className={ classnames(
+				'googlesitekit-wizard-progress-step',
+				`googlesitekit-wizard-progress-step--${ step }`,
+				{ 'googlesitekit-wizard-progress-step--current': currentStep },
+				`googlesitekit-wizard-progress-step--${ stepKey }`
```

Comment:
Let's put conditional objects last in the list 👍

Reply:
I agree it's not pretty. That's what I did initially but then realised, specificity going to change and it can cause problems

Reply:
Again, make sure to highlight this in a follow-up issue as it's unexpected.